website/source/docs/providers/aws/r/app_cookie_stickiness_policy.html.markdown

```diff
@@ -15,20 +15,20 @@ Provides an application cookie stickiness policy, which allows an ELB to wed its
 ```
 resource "aws_elb" "lb" {
     name = "test-lb"
-	  availability_zones = ["us-east-1a"]
-	  listener {
-	      instance_port = 8000
-		    instance_protocol = "http"
-		    lb_port = 80
-		    lb_protocol = "http"
-	  }
+	availability_zones = ["us-east-1a"]
+	listener {
+	  instance_port = 8000
+	  instance_protocol = "http"
+	  lb_port = 80
+	  lb_protocol = "http"
+    }
 }
 
 resource "aws_app_cookie_stickiness_policy" "foo" {
-	  name = "foo_policy"
-	  load_balancer = "${aws_elb.lb.id}"
-	  lb_port = 80
-	  cookie_name = "MyAppCookie"
+	name = "foo_policy"
```

Comment:
@radeksimko the validation rules for this are not actually noted anywhere

http://docs.aws.amazon.com/cli/latest/reference/elb/create-app-cookie-stickiness-policy.html

just says:

```
--policy-name (string)

The name of the policy being created. This name must be unique within the set of policies for this load balancer.
```

I am not sure where this error comes from. Even the go-sdk doesn't seem to list the specific error you are getting:

https://github.com/aws/aws-sdk-go/blob/master/apis/elasticloadbalancing/2012-06-01/api-2.json#L178